cw-orch-daemon/src/keys/private.rs

```diff
@@ -39,19 +39,22 @@ impl PrivateKey {
         secp: &Secp256k1<C>,
         coin_type: u32,
     ) -> Result<PrivateKey, DaemonError> {
-        let phrase = hkd32::mnemonic::Phrase::random(OsRng, hkd32::mnemonic::Language::English);
-
-        PrivateKey::gen_private_key_phrase(secp, phrase, 0, 0, coin_type, "")
+        Self::new_seed(secp, "", coin_type)
     }
     /// generate a new private key with a seed phrase
     pub fn new_seed<C: secp256k1::Signing + secp256k1::Context>(
         secp: &Secp256k1<C>,
         seed_phrase: &str,
         coin_type: u32,
     ) -> Result<PrivateKey, DaemonError> {
-        let phrase = hkd32::mnemonic::Phrase::random(OsRng, hkd32::mnemonic::Language::English);
-
-        PrivateKey::gen_private_key_phrase(secp, phrase, 0, 0, coin_type, seed_phrase)
+        let mut key = [0u8; 16];
+        OsRng.fill_bytes(&mut key);
+        match bip39::Mnemonic::from_entropy(&key) {
+            Ok(mnemonic) => {
+                PrivateKey::gen_private_key_phrase(secp, mnemonic, 0, 0, coin_type, seed_phrase)
+            }
+            Err(_) => Err(DaemonError::Phrasing),
```

Comment:
How can this ever err if menemonic is always created from a random key?

Reply:
I don't know but it returns a Result, so I have to check that. 
If it ever errors, it will output `Error`  which is not very good information, so I prefer Daemon error